src/select.js

```diff
@@ -0,0 +1,30 @@
+import Deferred from './deferred'
+
+function pairs ()
```

Comment:
oops, shouldn't have commited this line :(, thats what I get for just commiting stuff in a half complete state.